src/progressive-profiling/ProgressiveProfiling.jsx

```diff
@@ -171,6 +172,9 @@ const ProgressiveProfiling = (props) => {
 
   const handleSkip = (e) => {
     e.preventDefault();
+    if (nextUrl) {
+      window.location = nextUrl;
```

Comment:
why do we need to add it here?

Reply:
We also need to redirect to the next URL for the skip button click.